peps/pep-0785.rst

```diff
@@ -158,22 +169,23 @@ Working with exception groups is an intermediate-to-advanced topic, unlikely
 to arise for beginning programmers. We therefore suggest teaching this topic
 via documentation, and via just-in-time feedback from static analysis tools.
 In intermediate classes, we recommend teaching ``.leaf_exceptions()`` together
-with the ``.split()`` and ``.subgroup()`` methods, and mentioning
-``.preserve_context()`` as an advanced option to address specific pain points.
+with the ``.split()`` and ``.subgroup()`` methods, emphasizing a preference
+for the latter.
```

Comment:
Why do we need `leaf_exceptions()`? What can that do that ``.split()`` and ``.subgroup()`` cannot?